benchmark/url/url-searchparams-toString.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const common = require('../common.js');
+
+const values = {
+  noencode: {
```

Comment:
This does not benchmark different input types that a URLSearchParams can take

Reply:
Ok so adding values for string and iterable